index.html

```diff
@@ -0,0 +1,198 @@
+        <div class="address-txt">
+          <p><strong>Next Zoom conference(monday july 25/2022)</strong></p>
+          <p>@ BahirDar, Ethiopia</p>
+        </div>
+      </section>
+
+      <!-- Main Program -->
+
+      <section class="main-program">
+        <h2 class="main-pro">See The All Program</h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Coding</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Language</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-mobile fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Computer</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Programming</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-mobile fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Mobile</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <a href="#" class="service">See all services</a>
```

Comment:
- [ ] Kindly ensure that the `See All Services` link is not included in the `display:flex` of this section since it is causing it to wrap in weird ways on some screen sizes as you can see below.

![image](https://user-images.githubusercontent.com/90556221/181479662-c7fbad09-8229-48b0-b530-c37365b3fe48.png)

- [ ] Kindly also ensure that the text is aligned centre for the desktop version of your cards like in the Behance design.
![image](https://user-images.githubusercontent.com/90556221/181480308-5cfec5ca-a479-4afc-9e7f-96c01e6b9d73.png)